### PR TITLE
Gives SSdisease SS_NO_INIT

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

**Review by optimumtact — APPROVED** [review]

File: code/controllers/subsystem/disease.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SUBSYSTEM_DEF(disease)
 	name = "Disease"
-	flags = SS_NO_FIRE
+	flags = SS_NO_FIRE | SS_NO_INIT
 
 	var/list/active_diseases = list() //List of Active disease in all mobs; purely for quick referencing.
 	var/list/diseases
```
Comment (Fox-McCloud) [inline code/controllers/subsystem/disease.dm]: You may want to get rid of `archive_diseases` I forgot to do this.

**Comment by Fox-McCloud** [comment]
I had wondered if this was needed, but in all the hustle and bustle of changing everything and testing the diseases themselves, I forgot to ask about this. Oops.

**Comment by XDTM** [comment]
Archive diseases is pretty important atm, it's what keeps disease names

**Comment by optimumtact** [comment]
as evidenced by the compile failing

**Comment by lzimann** [comment]
@Cyberboss

**Pull request merged**